articles/quickstarts/qrng.md

```diff
@@ -48,3 +49,21 @@ Using the `H` operation, we are able to put our `Qubit` in superposition. To mea
 By putting our `Qubit` in superposition and measuring it, our result will be a different value each time the code is invoked. 
 
 When a `Qubit` is de-allocated it must be explicitly set back to the `Zero` state, otherwise the simulator will report a runtime error. An easy way to achieve this is invoking `Reset`.
+
+### Visualizating the code with the Bloch sphere
+
+In the Bloch sphere the north pole represents the classical value **0** and the south pole represents the classical value **1**. Any superposition can be represented by a point on the sphere (represented by an arrow). When the closer the end of the arrow to a pole, the higher the probability the qubit collapses into the classical value assigned to that pole when measured. For example, the qubit state represented by the red arrow below has a higher probability of giving the value **0** if we measure it.
+
+<img src="./Bloch.svg" width="175">
+
+We can use this representation to visualize what the code is doing:
+
+* First we start with a qubit initalizated in the state **0** and apply `H` to create a superposition in which the probabilities for **0** and **1** are the same.
+
+<img src="./H.svg" width="450">
+
+* Then we measure the qubit and save the output:
+
+<img src="./Measurement2.svg" width="450">
+
+Since the outcome obtained by the measurement is completely random we have obtained a random bit. We can call this function several times to create integers. For example, if we call the function three times to obtain three random bits, we can build random 3-bit numbers (this is, a random number between 1 and 8).
```

Comment:
```suggestion
Since the outcome obtained by the measurement is completely random we have obtained a random bit. We can call this function several times to create integers. For example, if we call the function three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
```

Reply:
I guess that my latent Matlab programmer took the control there.